app/common/data/models.py

```diff
@@ -162,6 +162,12 @@ class Collection(BaseModel):
     submission_period_start_date: Mapped[datetime.date | None] = mapped_column(nullable=True)
     submission_period_end_date: Mapped[datetime.date | None] = mapped_column(nullable=True)
 
+    @property
+    def has_submission_period_ended(self) -> bool:
+        if not self.submission_period_end_date:
+            return False
+        return self.submission_period_end_date < datetime.date.today()
```

Comment:
A little bit iffy here as we're ignoring timezones, _but_ it won't cause us a huge amount of trouble given that we're only dealing with GMT+BST. We should keep an eye on this carefully though.